client/my-sites/add-ons/hooks/use-add-on-display-cost.ts

```diff
@@ -12,16 +12,16 @@ const useAddOnDisplayCost = ( productSlug: string, quantity?: number ) => {
 		const product = getProductBySlug( state, productSlug );
 
 		if ( product?.product_term === 'month' ) {
+			/* Translators: $formattedCost: monthly price formatted with currency */
```

Comment:
It would be even better if we can adhere the same placeholder syntax, e.g. https://github.com/Automattic/wp-calypso/blob/trunk/client/components/marketing-survey/cancel-jetpack-form/jetpack-cancellation-offer.tsx#L71

```suggestion
			/* Translators: %(formattedCost)s: monthly price formatted with currency */
```